Formula/tendermint.rb

```diff
@@ -0,0 +1,22 @@
+class Tendermint < Formula
+  desc "BFT state machine replication for applications across all programming languages"
+  homepage "https://tendermint.com/"
+  url "https://github.com/tendermint/tendermint/archive/v0.34.0.tar.gz"
+  sha256 "3a28fac4c5e610fc32763db1b717ec0a1e12d39262e321d6da223c3b0acfea7f"
+  license "Apache-2.0"
+  head "https://github.com/tendermint/tendermint.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build", "LDFLAGS=-X github.com/tendermint/tendermint/version.TMCoreSemVer=#{version}"
```

Comment:
```suggestion
    system "make", "build", "VERSION=#{version}"
```

Can you try this?

Reply:
Yup. It seems to also work. Thanks for that